src/services/schemas/properties/SchemaPropertyService.ts

```diff
@@ -106,10 +106,6 @@ export abstract class SchemaPropertyService {
     const defaults = getSchemaPropertyDefaultValidators(this.property, required)
     const validators = fieldRules(title, ...this.validators)
 
-    if (this.componentIs(JsonInput)) {
-      validators.push(isJson(title))
-    }
-
```

Comment:
Has this ever been part of the problem? If the value from the input itself is not json the mappers are likely to fail and post a valid json string to the server. Which means errors at runtime rather than at form submission.

Reply:
Yeah this is something I'd like to address longer-term. Because of the way we're mapping request values, JSON inputs would require that bare text be wrapped in quotes to be valid (according to this validator, which is technically correct). However, that means that when the values are sent to the server, they're stringified, which means this: `"test"` becomes `"\"test\""` - which is obviously incorrect. The obvious solution, which is what we were doing before, is to parse that value on the way out so that `"\"test\""` is unwrapped to `"test"`; the problem with that is that we're feeding those request values directly back into the form and parsing them again, which is why quotes were disappearing.

Reply:
Ah okay makes sense.